yellowbrick/classifier/threshold.py

```diff
@@ -0,0 +1,351 @@
+    """Quick method for ThresholdVisualizer.
+    Visualizes precision, recall and queue rate at different different
+    thresholds for binary targets.
+
+    Parameters
+    ----------
+
+    model : a Scikit-Learn classifier, required
+        Should be an instance of a classifier otherwise a will raise a
+        YellowbrickTypeError exception on instantiation.
+
+    colormap : string or cmap, default: None
+        Optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    color : String, default: None
+        Optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    n_trials : Integer, default: 50
+        Number of trials to conduct via train_test_split
+
+    quantiles : Sequence, default: (0.1, 0.5, .9)
+        Setting the quantiles for visualizing model variability using
+        scipy.stats.mstats.mquantiles
+
+    random_state : Integer, default: 0
+        Random state integer for sampling in train_test_split
+
+    title : String, default: None
+        Title of the visualization
+
+    kwargs : keyword arguments passed to the super class.
+
+    Returns
+    -------
+    ax : matplotlib axes
+        Returns the axes that the parallel coordinates were drawn on.
+    """
+    # Instantiate the visualizer
+    visualizer = ThresholdVisualizer(
+        model,
+        colormap=colormap,
+        color=color,
+        n_trials=n_trials,
+        test_size_percent=test_size_percent,
+        quantiles=quantiles,
+        random_state=random_state,
+        title=title,
+        **kwargs)
+
+    # Fit and transform the visualizer (calls draw)
+    visualizer.fit_draw_poof(X, y)
+
+    # Return the axes object on the visualizer
+    return visualizer.ax
+
+
+##########################################################################
+# Static ThresholdVisualizer Visualizer
+##########################################################################
+
+
+class ThresholdVisualizer(ModelVisualizer):
+    """Visualizes precision, recall and queue rate at different different
+    thresholds for binary targets.
+
+    Parameters
+    ----------
+
+    model : a Scikit-Learn classifier, required
+        Should be an instance of a classifier otherwise a will raise a
+        YellowbrickTypeError exception on instantiation.
+
+    colormap : string or cmap, default: None
+        Optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    color : String, default: None
+        Optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
```

Comment:
So I think color vs colormap is a hold over from Pandas (and not the current version) where either a color or colormap could be passed in. This made a lot of sense for feature visualizers where the target could either be discrete (a classifier or clusterer) and therefore required a color cycle or discrete list of colors or it could be continuous (a regression) and a colormap was required. 

So here are Ben's (possibly erroneous) notes on color and `resolve_colors()` as I can think of them off the top of my head:

1. `color` refers to a discrete sequence of colors 
    1. color can be a string, a tuple, a float, or a list of strings, list of floats or a list of tuples 
    2. in most cases, it should be a list of strings or a list of tuples 
    3. a float indicates a grayscale value 
    4. a tuple indicates an RGB/A  value 
    5. a string can be either a color name or reference, e.g. `bgrmcyk` 
    6. in the case of lists, a string can also be the name of a palette, e.g. `"Set1"`
2. `colormap` refers to a continuous sequence of colors 
    1. colormap can be a string or a collection 
    2. a string is the name of a colormap, e.g. `"RdBu"`) 
    3. a string with an `_r` at the end reverses the colormap, e.g. `"RdBu_r"`. 
    4. a string can be the name of a palette, which is converted to a `ListedColorMap` 
    4. a collection of colors is converted into a `ListedColorMap` 
3. Because we need to get a distinct n - # of colors two things happen:
    1. We get a slice of discrete colors passed to `color` or a cycle for n > len(colors), repeating colors as we go. 
    2. For a colormap, n determines the exact color values in the heatmap
4. With a colormap, we need to add a `colorbar`, with `color` we need a legend. 

Ok, so that's a start of my thoughts - is this jibing more or less?